ts/Core/Series/SeriesDefaults.ts

```diff
@@ -1951,13 +1951,14 @@ const seriesDefaults: SeriesOptions = {
         x: 0,
 
         /**
-         * The Z index of the data labels. The default Z index puts it above
-         * the series. Use a Z index of 2 to display it behind the series.
+         * The z index of the data labels.
+         * Use a zIndex of 6 to display it above the series,
+         * or use a zIndex of 2 to display it behind the series.
```

Comment:
1. No line wraps without a paragraph, [see the style guide](https://github.com/highcharts/highcharts/blob/master/repo-guidelines.md#1-avoid-line-breaks-without-paragraphs).
2. Also, zIndex is code, so it should be in back ticks `zIndex`.